contracts/mixnet/artifacts/checksums.txt

```diff
@@ -0,0 +1 @@
+d8a3bddfd2d9f530ca373dfadf60a83eb1a199febec596a3ed37024a22012767  mixnode.wasm
```

Comment:
Not sure how to leave comment on a binary file, but this related to another artifact - should `contracts/mixnet/artifacts/mixnode.wasm` be commited to the repo?